include/net/net_context.h

```diff
@@ -293,6 +293,15 @@ struct net_context {
 
 	/** Network interface assigned to this context */
 	s8_t iface;
+
+
+	/** IPv6 hop limit or IPv4 ttl for this network packet.
```

Comment:
Do we really need to store this in net_context? The hop limit/ttl is already stored in net_if and we have interface index in net_context so we could fetch it from net_if directly.

Reply:
What about temporary ttl/hop_limit tweak? As far as I can see, that's what is happening in resolve.c

Reply:
Yep, you are right here, I missed the temp hop-limit setting in LLMNR

Reply:
Indeed, quickly  c/p from net_pkt